tests/virus_total_agent_test.py

```diff
@@ -71,6 +71,82 @@ def virustotal_valid_response(message: msg.Message) -> dict[str, Any]:
     return response
 
 
+def virustotal_secure_valid_response(message: msg.Message) -> dict[str, Any]:
+    """Method for mocking the Virus Total public API valid response."""
+    del message
+    response = {
+        "results": {
+            "scans": {
+                "Bkav": {
+                    "detected": False,
+                    "version": "1.3.0.9899",
+                    "result": None,
+                    "update": "20220107",
+                },
+                "Elastic": {
+                    "detected": False,
+                    "version": "4.0.32",
+                    "result": "eicar",
+                    "update": "20211223",
+                },
+            },
+            "scan_id": "ID42",
+            "sha1": "some_sha1",
+            "resource": "some_ressource_id",
+            "response_code": 1,
+        },
+        "response_code": 200,
+    }
+    return response
+
+
+def virustotal_unreliable_scanner_response(message: msg.Message) -> dict[str, Any]:
```

Comment:
Why not declare this as fixture?

Reply:
its the style present on the agent

Reply:
Gradually improve. start by this, so it will be easier for the next person.
However, I would put the response as constant, and create an inner function for the side-effect in the unit test.